samples/matter/thermostat/src/temperature_manager.cpp

```diff
@@ -0,0 +1,121 @@
+/*
+ * Copyright (c) 2023 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#include "temperature_manager.h"
+#include "app_config.h"
+#include "app_task.h"
+#include <zephyr/logging/log.h>
+
+LOG_MODULE_DECLARE(app, CONFIG_CHIP_APP_LOG_LEVEL);
+
+using namespace chip;
+using namespace ::chip::DeviceLayer;
+
+constexpr EndpointId kThermostatEndpoint = 1;
+
+namespace ThermostatAttributes = chip::app::Clusters::Thermostat::Attributes;
```

Comment:
What about `using chip::app::Clusters::Thermostat::Attributes` instead of creating a new namespace? But if you want to keep the code more readable by doing it this way, I'm ok with it :)